tests/c/go.mod

```
@@ -0,0 +1,3 @@
+module tests/c
+
+go 1.22.1
```

Don't think patch is required, can we keep major/minor?

```suggestion
go 1.22
```